login-workflow/src/new-architecture/screens/SuccessScreen/SuccessScreenBase.tsx

```diff
@@ -15,7 +24,9 @@ export const SuccessScreenBase: React.FC<SuccessScreenProps> = (props) => {
         <WorkflowCard {...cardBaseProps}>
             <WorkflowCardHeader {...headerProps} />
             <WorkflowCardBody>
-                <WorkflowFinishState icon={icon} title={messageTitle} description={message} />
+                <ErrorManager {...errorDisplayConfig}>
+                    <WorkflowFinishState icon={icon} title={messageTitle} description={message} />
+                </ErrorManager>
```

Comment:
Why is there an error manager in here? This is a success screen... errors should be handled before we even get to this screen.